### DISCUSSION TITLE
Calculation method for Token Usage

### DISCUSSION THREAD

**Discussion by secsilm** [langgenius/dify#3516] [Help]
Is Token Usage the total of prompt+completion? Where can I set the price?

![image](https://github.com/langgenius/dify/assets/8894643/fbc7705e-cb5f-4e4e-8468-6a2165f307be)

**Comment by guchenhe** [comment]
This is the sum total consumption. 

the calculation algorithm depends on the underlying model, in some cases it might be an estimation than an exact calculation. we currently do not support price limit / alerts.
